client/src/pages/QuizPage.jsx

```diff
@@ -1,10 +1,10 @@
-import React, { useState, useEffect } from "react";
+import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 import axios from "axios";
-import Footer from "../components/footer/footer";
-import Header from "../components/header/index";
+import Footer from "../components/footer";
+import Header from "../components/header";
 import Success from "../components/success";
-import Option from "../components/option-botton";
+import OptionButton from "../components/OptionButton/OptionButton";
```

Comment:
it should be 
`import OptionButton from "../components/OptionButton";`

Reply:
Changed to requested format

Reply:
Update as requested